app/schema_validation/__init__.py

```diff
@@ -16,13 +18,11 @@ def build_error_message(errors, schema):
         field = "'{}' {}".format(e.path[0], e.schema.get('validationMessage')) if e.schema.get(
             'validationMessage') else e.message
         s = field.split("'")
-        field = {s[1]: s[2].strip()}
+        field = OrderedDict({"error": "ValidationError", "message": "{}{}".format(s[1], s[2])})
```

Comment:
OrderedDict doesn't gain us anything here - especially in its current invocation, which would simply keep the ordering of the dict containing `error` and `message`, which as we know is arbitrary.

kill the ordered dict i say